components/developer/rust/Makefile

```diff
@@ -114,27 +112,21 @@ CONFIGURE_OPTIONS += --enable-llvm-link-shared
 CONFIGURE_OPTIONS += --llvm-root=$(CLANG_PREFIX)
 CONFIGURE_OPTIONS += --llvm-config=$(CLANG_PREFIX)/bin
 
-COMPONENT_BUILD_CMD = $(PYTHON) x.py build 
-COMPONENT_INSTALL_CMD = $(PYTHON) x.py install
-COMPONENT_INSTALL_ARGS =
-
 # Cleanup standard environment!
 COMPONENT_BUILD_ENV =
 COMPONENT_BUILD_ENV += OPENSSL_DIR=$(OPENSSL_PREFIX)
 COMPONENT_BUILD_ENV += OPENSSL_LIB_DIR=$(OPENSSL_LIBDIR)
 COMPONENT_BUILD_ENV += OPENSSL_INCLUDE_DIR=$(OPENSSL_INCDIR)
-COMPONENT_BUILD_ENV += OPENSSL_STATIC=0
 COMPONENT_BUILD_ENV += CC=$(CC)
 COMPONENT_BUILD_ENV += CFLAGS="$(CFLAGS)"
 COMPONENT_BUILD_ENV += CXX=$(CXX)
 COMPONENT_BUILD_ENV += CXXFLAGS="$(CXXFLAGS)"
 COMPONENT_BUILD_ENV += AR=$(GNUAR)
 COMPONENT_BUILD_ENV += RUSTC=/usr/bin/rustc
 COMPONENT_BUILD_ENV += CARGO_HOME=$(CARGO_HOME)
-COMPONENT_BUILD_ENV += GMAKE=
-# Enforce linker consistency
-COMPONENT_BUILD_ENV +=	RUSTFLAGS="-C linker=$(CC)"
-COMPONENT_BUILD_ENV +=	RUST_BACKTRACE=1
+COMPONENT_BUILD_ENV += PATH=$(PATH)
```

Comment:
This should be not needed, see https://github.com/OpenIndiana/oi-userland/blob/b6799ea75360955c287ccc8757c1d9d92fc68abf/make-rules/shared-macros.mk#L1389